apps/web/src/components/Advisory.tsx

```diff
@@ -0,0 +1,13 @@
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const Advisory: React.FC = ({ children }) => {
```

Comment:
Can we call this `Alert` to match the BC gov term? https://developer.gov.bc.ca/Design-System/Alert-Banners

Reply:
Sure, but I feel weird about it for a couple reasons. 
1. Alert is already an html tag name, and does something very different 
2. The BC gov alert component has a lot more features/functions that what this component does. It feels like either a mismatched component or a fair bit of scope creep.

Reply:
It's just different variants and accessibility features isn't it? I think we should implement as much as is reasonable, accessibility stuff is pretty important but we don't need the other variants for now (warning/error/etc..)